strax/corrections.py

```diff
@@ -38,31 +47,38 @@ def __init__(self, host='127.0 0.1', username=None, password=None,
         self.database = self.client[self.database_name]
 
     def list_corrections(self):
-        '''Smart logic to list all corrections.
-        '''
+        """Smart logic to list all corrections.
+        """
         return [x['name'] for x in self.database.list_collections()]
 
     def read(self, correction):
-        '''Smart logic to read corrections,
-        :param correction: correction's name (str type).
-        '''
+        """Smart logic to read corrections,
+        :param correction: pandas.DataFrame object name in the DB (str type).
+        """
         df = pdm.read_mongo(correction, [], self.database)
 
         # No data found
         if df.size == 0:
             return None
         # Delete internal Mongo identifier
         del df['_id']
+
+        df['time'] = pd.to_datetime(df['time'], utc=True)
+
         return df.set_index('time')
 
-    def interpolate(self, what, when, how='interpolate', **kwargs):
-        '''Interpolate values of a given correction
+    @staticmethod
```

Comment:
Sorry Aaron, this suggestion is no longer valid of you need ```self.check_timezone```. You can remove this line and use self as a first argument in the function